src/components/ec-multiple-values-selection/ec-multiple-values-selection.vue

```diff
@@ -0,0 +1,275 @@
+          :key="item.value"
+          class="ec-multiple-values-selection__value-wrapper"
+          data-test="ec-multiple-values-selection__checkbox-deselect"
+        >
+          <ec-checkbox
+            checked
+            class="ec-multiple-values-selection__checkbox"
+            @checked-value-change="onDeselect(item)"
+          >
+            <template #label>
+              <div
+                class="ec-multiple-values-selection__label-wrapper"
+              >
+                <ec-icon
+                  v-if="item.icon"
+                  class="ec-multiple-values-selection__icon"
+                  :name="item.icon.name"
+                  :type="item.icon.type"
+                  :size="24"
+                />
+                <span class="ec-multiple-values-selection__label-text">{{ item.text }}</span>
+              </div>
+            </template>
+          </ec-checkbox>
+        </li>
+
+        <li
+          v-for="item in notSelectedItems"
+          :key="item.value"
+          class="ec-multiple-values-selection__value-wrapper"
+          data-test="ec-multiple-values-selection__value-wrapper"
+        >
+          <ec-checkbox
+            class="ec-multiple-values-selection__checkbox"
+            data-test="ec-multiple-values-selection__checkbox-select"
+            @checked-value-change="onSelect(item)"
+          >
+            <template #label>
+              <div
+                class="ec-multiple-values-selection__label-wrapper"
+              >
+                <ec-icon
+                  v-if="item.icon"
+                  class="ec-multiple-values-selection__icon"
+                  :name="item.icon.name"
+                  :type="item.icon.type"
+                  :size="24"
+                />
+                <span class="ec-multiple-values-selection__label-text">{{ item.text }}</span>
+              </div>
+            </template>
+          </ec-checkbox>
+        </li>
+      </div>
+    </ec-loading>
+  </div>
+</template>
+<script>
+// TODO ONL-4912 !!!don't forget to debounce search!!!
+import EcCheckbox from '../ec-checkbox';
+import EcIcon from '../ec-icon';
+import EcInputField from '../ec-input-field';
+import EcLoading from '../ec-loading';
+
+export default {
+  name: 'EcMultipleValuesSelection',
+  components: {
+    EcCheckbox, EcIcon, EcInputField, EcLoading,
+  },
+  model: {
+    prop: 'value',
+    event: 'change',
+  },
+  props: {
+    items: {
+      type: Array,
+      required: true,
+      default: () => ([]),
+    },
+    value: {
+      type: Array,
+      required: false,
+      default: () => ([]),
+    },
+    loading: {
+      type: Boolean,
+    },
+    error: {
+      type: Error,
+      required: false,
+      default: null,
+    },
+    emptyMessage: {
+      type: String,
+    },
+    emptyIcon: {
+      type: String,
+    },
+    isSearchable: {
+      type: Boolean,
+    }, // TODO with https://fxsolutions.atlassian.net/browse/ONL-4912
+    canSelectAll: {
```

Comment:
Maybe something with "is" since a boolean like the above "isSearchable"?

Reply:
while I usually agree, what would this be? `isSelectAllable` on the other hand we need to keep sane naming and a bit of `canSomething` won't hurt and improve readability? Other thoughts?

Reply:
isSelectable? is english?